src/main/java/io/jenkins/plugins/audit/listeners/UserLogListener.java

```diff
@@ -22,14 +22,16 @@
 
 package io.jenkins.plugins.audit.listeners;
 
-import org.apache.logging.log4j.audit.LogEventFactory;
-
-import hudson.ExtensionList;
 import hudson.Extension;
+import hudson.ExtensionList;
 import io.jenkins.plugins.audit.event.Login;
 import io.jenkins.plugins.audit.event.Logout;
-import javax.annotation.Nonnull;
 import jenkins.security.SecurityListener;
+import org.apache.logging.log4j.audit.LogEventFactory;
+
+import javax.annotation.Nonnull;
```

Comment:
AuditLogConfiguration uses `edu.umd.cs.findbugs.annotations.NonNull`. Is there a reason for the difference?

Reply:
Different class file authors probably?

Reply:
Updated everything to converge on the findbugs package.